docs/arlas-ogc.md

```diff
@@ -15,7 +15,7 @@ The table below lists the URL endpoints and their ogc "parameters". The paramete
 | ---------------------------------------- | ---------------------------------------- |
 | /arlas/ogc/wfs/`{collection}`?`wfsParameters`      | Download the features of the given `{collection}` |
 | /arlas/ogc/csw/?`cswParameters`      | Discover the available collections in ARLAS |
-| /arlas/ogc/csw/opensearch      | OpenSearch CSW Description Document |
```

Comment:
Same there is two endpoints one in `arlas/ogc` and the other in `arlas/explore` : 
- ` /arlas/explore/ogc/opensearch/{collection}`
- `/arlas/ogc/csw/opensearch`